roles/matrix-bridge-appservice-slack/defaults/main.yml

```diff
@@ -0,0 +1,92 @@
+# matrix-appservice-slack is a Matrix <-> Slack bridge
+# See: https://github.com/matrix-org/matrix-appservice-slack
+
+matrix_appservice_slack_enabled: true
+
+matrix_appservice_slack_docker_image: "cadair/matrix-appservice-slack:latest"
+
+matrix_appservice_slack_base_path: "{{ matrix_base_data_path }}/appservice-slack"
+
+# Once you make a control room in Matrix, you can get its ID by typing any message and checking its source
+slack_control_room_id: ''
+matrix_appservice_slack_bot_name: 'slackbot'
+
+# Controls the SLACK_PORT and MATRIX_PORT of the installation
+matrix_appservice_slack_matrix_port: 8008
+matrix_appservice_slack_slack_port: 9003
+
+matrix_appservice_slack_homeserver_address: 'http://matrix-synapse:{{ matrix_appservice_slack_matrix_port }}'
```

Comment:
Renaming `matrix_appservice_slack_homeserver_address` to `matrix_appservice_slack_homeserver_url` is more consistent with how we use it in the configuration below.